src/blurredLocationDisplay.js

```diff
@@ -7,11 +7,35 @@ BlurredLocationDisplay = function BlurredLocationDisplay(options) {
 
   options.Interface = options.Interface || require('./ui/Interface.js');
   options.blurredLocation = options.blurredLocation || {};
+  options.PLpeopleAPI = options.PLpeopleAPI || false ;
```

Comment:
I guess i'm not sure we should customize PL.org completely into here - perhaps better to pass in a custom parser for PL.org API in the example and docs, but not build it into the general-purpose library? What do you think?